docs/ref/contrib/syndication.txt

```diff
@@ -955,6 +979,13 @@ They share this interface:
     * ``updateddate`` should be a Python  :class:`~datetime.datetime` object.
     * ``enclosure`` should be an instance of
       :class:`django.utils.feedgenerator.Enclosure`.
+
+      .. deprecated:: 1.9
```

Comment:
Add blank line after `.. deprecated:: 1.9` and move this section after the list and say, "The ``enclosures`` keyword argument ..."